libevmcore/EVMSchedule.h

```diff
@@ -63,11 +63,34 @@ struct EVMSchedule
 	unsigned txDataZeroGas = 4;
 	unsigned txDataNonZeroGas = 68;
 	unsigned copyGas = 3;
+
+	//EIP150
+	unsigned extcodesizeGas = 20;
+	unsigned extcodecopyGas = 20;
+	unsigned balanceGas = 20;
+	unsigned suicideGas = 0;
+
+	bool isEIP150() const { return callGas == 700; }
```

Comment:
I would avoid having redundant data in the schedule struct. But you suggestion looks fine. I would be even better to have `unsinged callDepthLimit` equal 1024 or 0. Now we have this magic number in many places. But this should not go with this PR.